subscription-manager-cockpit.spec.tmpl

```diff
@@ -3,7 +3,7 @@ Version: %{VERSION}
 Release: 1%{?dist}
 Summary: Subscription Manager Cockpit UI
 Group: System Environment/Base
-License: GPLv2
+License: LGPLv2
```

Comment:
Why do you want to change license?

Reply:
It is explained in the commit message:
> The cockpit plugin was already licensed as LGPL-2.1, hence fix the
> License tag in the RPM packaging (small mistake I did when creating the
> new packaging).

Reply:
> current license of subscription-manager-cockpit-plugin RPM is GPLv2 despite LICENSE file contains LGPLv2 license

This is exactly what one of the commits in PR is supposed to do: fix the wrong license tag in the packaging. :) `license` in `package.json` & the `LICENSE` files were already OK, it was only a mistake on my side when I added the RPM packaging after the split. The code is already LGPL v2.